source_pubsub/client.go

```diff
@@ -145,10 +151,13 @@ func (c *PubSubClient) Initialize(ctx context.Context) error {
 		return err
 	}
 
-	c.tomb.Go(func() error {
-		ctx := c.tomb.Context(nil) //nolint:staticcheck // SA1012 tomb expects nil
-		return c.startCDC(ctx)
-	})
+	for _, topic := range c.topicNames {
+		c.tomb.Go(func() error {
+			ctx := c.tomb.Context(nil) //nolint:staticcheck // SA1012 tomb expects nil
+			c.startCDC(ctx, topic)
+			return nil
```

Comment:
@anna-cross is it intended to not return the result of `c.startCDC()`? if so, what's the context?

My understanding is that this is anyways within the `func()` passed in above, so it shouldn't break the for loop

Reply:
Woops yes! I'll move it back into return